CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Master (Unreleased)
 
+## 1.36.0 (2019-09-25)
+
 * Fix `RSpec/DescribedClass`'s error when a `described_class` is part of the namespace. ([@pirj][])
```

Comment:
[Bad copy-pasta](https://github.com/rubocop-hq/rubocop-rspec/commit/11659ced7dde8e2ff19efa8debc306de3a0d6e80#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR5), the description is a duplicate of what has been fixed in 1.34.1. Sorry about that.
Should be:
> Fix `RSpec/DescribedClass`'s error when `described_class` is used as part of a constant. ([@pirj][])

Reply:
I’ll fix it.